src/main/java/net/dries007/tfc/objects/blocks/devices/BlockFirePit.java

```diff
@@ -274,25 +274,14 @@ else if (tile.getCookingPotStage() == TEFirePit.CookingPotStage.FINISHED)
         return true;
     }
 
-    @Override
-    public void onEntityWalk(World worldIn, BlockPos pos, Entity entityIn)
-    {
-        IBlockState state = worldIn.getBlockState(pos);
-        if (state.getValue(LIT) && !entityIn.isImmuneToFire() && entityIn instanceof EntityLivingBase)
-        {
-            entityIn.attackEntityFrom(DamageSource.IN_FIRE, 1.0F);
-        }
-        super.onEntityWalk(worldIn, pos, entityIn);
-    }
-
```

Comment:
This was intentionally added to fix #1025, which the result behavior was that fire pits only damaged you in very select circumstances. Is the problem not the below change to the wrong `super` call?

Reply:
Oh, no that wrong super call did nothing. Looking at the call chain the default implementation of both methods does nothing. Now, removing `#onEntityWalk` fixed. 

Looking at the call chain, `#onEntityWalk` is called from `Entity#move()L1039` when said entity is in the block or one block above it if the above block is empty/AIR, so, if there is no other way to fix #1025 we're gonna have to choose which one stays fixed.

Reply:
Removed fix as per discord discussion